### ISSUE TITLE
creating the environment

### ISSUE THREAD

**Issue by CLG68** [molecularmodelinglab/banana#3]
Hi,

You just changed the requirements for the environment.  However, awandb does not exist, it should be changed back to wandb

In the requirements, only terrace needs to be at 0.0.6.  The other ones could be up-to date.
Here is a combination that worked for me:

conda install conda-forge::wandb
conda install conda-forge::biopandas
conda install conda-forge::joblib
conda install conda-forge::tqdm
conda install conda-forge::biopython
conda install conda-forge::scipy
conda install conda-forge::dgllife
conda install conda-forge::sympy
pip install git-timewarp
conda install conda-forge::seaborn
pip install terrace==0.0.6
pip install argparse
conda install conda-forge::omegaconf
conda install nvidia/label/cuda-12.1.0::cuda-toolkit
pip install cuda-python==12.1
pip install torch torchvision torchaudio
pip install pytorch-lightning
conda install conda-forge::torchmetrics
pip install torchdata
pip install  dgl -f https://data.dgl.ai/wheels/cu121/repo.html
pip install  dglgo -f https://data.dgl.ai/wheels-test/repo.html
conda install rdkit

it is best, after the environment is created, to test cuda support for torch as this is usually the problem when it does not work.  

$ python
>>> import torch
>>> torch.cuda.is_available()
True
>>> exit ()

If it does not indicate True, you could always force it to 12.1 using 
conda install pytorch torchvision torchaudio pytorch-cuda=12.1 -c pytorch -c nvidia
